oojs.js

```diff
@@ -10,3 +10,27 @@
   // Have an artist property that is determined by some input.
 
 // Create an instance of the Album class.
+
+class Playlist {
+  constructor (title, songs) {
+    this.title = title;
+    this.songs = ["Workout", "Fun", "Zumba"];
```

Comment:
You don't want to hardcode an array of songs here. Use the `songs` argument you're already passing into the constructor.

`this.songs = songs`